infra/feast-operator/api/v1alpha1/featurestore_types.go

```diff
@@ -160,7 +159,7 @@ type OnlineStoreFilePersistence struct {
 
 // OnlineStoreDBStorePersistence configures the DB store persistence for the online store service
 type OnlineStoreDBStorePersistence struct {
-	// Type of the persistence type you want to use. Allowed values are: snowflake.online, redis, ikv, datastore, dynamodb, bigtable, postgres, cassandra, mysql, hazelcast, singlestore, hbase, elasticsearch, qdrant, couchbase, milvus
+	// Type of the persistence type you want to use.
```

Comment:
why this comments section removed

Reply:
i guess i was concerned with it getting out of date as there's not way to test that its accurate.

i've added a test to ensure the next line, kubebuilder's enum validation, is accurate... and so i guess i felt like that should be sufficient as it will inform the user which data sources are supported.

that said, if we think the comment is important i can add it back.

Reply:
I saw similar comments are there for`OfflineStoreDBStorePersistence`  wether we keep both or removed from OfflineStoreDBStorePersistence too WDYT ?

Reply:
Yes fine with we can remove due to out of date issue.